### PR TITLE
add tigris tls renewal steps

### PR DESCRIPTION
**Pull request by kayiwa**

### CODE REVIEW

File: services/create_ssl_certs.md [hunk 1]
```diff
@@ -170,5 +173,22 @@ Submit the CSR to gandi.net. Use the following guidance:
...
+  cat ~/path/to/downloads/tigris_princeton_edu_cert.cer ~/path/to/downloads/tigris_princeton_edu_interm.cer > keys/tigris_princeton_edu_chained.pem
+  ```
+
+This will generate a chained file. You will be prompted for a password in the next step.
+
+```bash
+  openssl pkcs12 -export -out tigris_princeton_edu.pfx -inkey tigris_princeton_edu_priv.key -in tigris_princeton_edu_chained.pem
+```
+
+Send the resulting file to the tigris support folks via [the Secure Send Portal](https://securesend.princeton.edu/#/) along with the password used above
```
Comment (kevinreiss) [inline services/create_ssl_certs.md:190]: Maybe rewrite this with a first step to a open a ticket. I did this and then they gave me an individual email they wanted the securesend to go to. 

Suggestion - 

Open a ticket with tigris support at Gimmal Support support@gimmal.com and ask who should receive the new chained file. Send the resulting file to the email tigris support recommends to via [the Secure Send Portal](https://securesend.princeton.edu/#/) along with the password used above.

**Review by kevinreiss — COMMENTED** [review]
Looks good Francis. Just one clarification on the process of submitting the updated cert file to the vendor.

**Review by kevinreiss — DISMISSED** [review]
See my comment clarifying the steps to submit the "new" file.

File: services/create_ssl_certs.md (hunk 1 above)
Comment (acozine) [reply services/create_ssl_certs.md:192]: Addressed by a2426f4554d2bdb6771503f5bf0cda56093b53ba.

**Review by acozine — APPROVED** [review]
Looks good, thanks @kayiwa

**Pull request merged**